presets/force-stable-fedora.json

```diff
@@ -0,0 +1,33 @@
+// This preset is used to force Renovate to only suggest updates to stable Fedora releases
```

Comment:
1. Could you rename this file with `.json5` suffix?
2. I'd suggest place the new preset under `renovate/presets` directory, or maybe `config/renovate/presets` since some teams are referencing the `config/renovate/renovate.json` , putting them in the same `config` dir seems make more sense, wdyt?

Reply:
Agree with both, thank you. (Not sure why vscode even allowed me the first point :) )